apps/trading/client-pages/portfolio/withdrawals-container.tsx

```diff
@@ -45,17 +44,13 @@ export const WithdrawalsContainer = () => {
           <div className="w-full dark:bg-black bg-white absolute bottom-0 h-auto flex justify-end px-[11px] py-2">
             <Button
               size="sm"
-              onClick={() => setWithdrawDialog(true)}
+              onClick={() => openWithdrawDialog()}
```

Comment:
suggestion (non blocking): no need for new function
```suggestion
              onClick={openWithdrawDialog}
```

Reply:
Resolving as onClick handler args dont fulfill function signature of openWithdrawDialog